Application/EdFi.Ods.AdminApi/appsettings.json

```diff
@@ -18,8 +18,8 @@
     },
     "EnableDockerEnvironment": false,
     "ConnectionStrings": {
-        "EdFi_Admin": "Data Source=.\\;Initial Catalog=EdFi_Admin;Integrated Security=True",
-        "EdFi_Security": "Data Source=.\\;Initial Catalog=EdFi_Security;Integrated Security=True"
+        "EdFi_Admin": "Data Source=.\\;Initial Catalog=EdFi_Admin;Integrated Security=True;Encrypt=False",
+        "EdFi_Security": "Data Source=.\\;Initial Catalog=EdFi_Security;Integrated Security=True;Encrypt=False"
```

Comment:
Please make this change in the development appsettings - the real appsettings file should _not_ disable encryption by default.